README.md

```diff
@@ -212,7 +212,8 @@ API | Description | Auth | HTTPS | CORS | Link |
 | DomainDb Info | Domain name search to find all domains containing particular words/phrases/etc | No | Yes | Unknown | [Go!](https://domainsdb.info/apidomainsdb/index.php) |
 | Faceplusplus | A tool to detect face | `OAuth` | Yes | Unknown | [Go!](https://www.faceplusplus.com/) |
 | Genderize.io | Determines a gender from a first name | No | Yes | Unknown | [Go!](https://genderize.io) |
-| Github | Information for a user's Github profile | No | Yes | Unknown | [Go!](https://developer.github.com/v3/) |
+| Github | GitHub REST API v3 | `OAuth` | Yes | Unknown | [Go!](https://developer.github.com/v3/) |
```

Comment:
Please use a more appropriate description. Also, this service does support CORS.